cider/exec/nextgen/operators/FilterNode.h

```diff
@@ -39,23 +34,25 @@ class FilterNode : public OpNode {
     (exprs_.emplace_back(std::forward<T>(exprs)), ...);
   }
 
-  std::vector<ExprPtr> exprs_;
+  ExprPtrVector getExprs() override { return exprs_; }
+
+  ExprPtrVector exprs_;
```

Comment:
Why this member `exprs_` is also public?

Reply:
Will be refactored in the future.